src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -329,28 +333,12 @@ export const selectPortfolioAccountIdsSortedFiat = createSelector(
   selectPortfolioTotalFiatBalanceByAccount,
   selectAssets,
   (totalAccountBalances, assets) => {
-    const sortedAccountBalances = Object.entries(totalAccountBalances)
-      .sort(([_, accountBalanceA], [__, accountBalanceB]) =>
-        bnOrZero(accountBalanceA).gte(bnOrZero(accountBalanceB)) ? -1 : 1
-      )
-      .map(([accountId, _]) => accountId)
-
-    const sortedAccountBalancesByChainBuckets = sortedAccountBalances.reduce(
-      (acc: Record<ChainTypes, CAIP10[]>, accountId) => {
-        const assetId = accountIdToFeeAssetId(accountId)
-        const asset = assets[assetId]
-
-        if (!acc[asset.chain]) {
-          acc[asset.chain] = []
-        }
-
-        acc[asset.chain] = [...acc[asset.chain], accountId]
-        return acc
-      },
-      {} as Record<ChainTypes, CAIP10[]>
+    const sortedAccountBalances = makeSortedAccountBalances(totalAccountBalances)
```

Comment:
can't comment on the line above, but any selector logic that finishes with `map` or `filter` should probably be using the deep equal output selectors, and `createSelector`, as they're not referentially stable, even though they'll output the same ids